include/ews/ews.hpp

```diff
@@ -17385,6 +17436,19 @@ class basic_service final
         return *this;
     }
 
+    //! Synchronizes a folder in the Exchagne store.
+    sync_folder_items_result sync_folder_items(
+        const folder_id& folder_id,
+        const std::string& sync_state = "",
+        int max_changes_returned = 512,
+        std::vector<item_id> ignored_items = {})
+    {
+        return sync_folder_items_impl(folder_id,
+            sync_state,
+            max_changes_returned,
+            ignored_items);
+    }
```

Comment:
```c++
std::vector<item_id> ignored_items = {}
```
This won't compile with VS 2012 because ancient VS 2012 does not support initializer lists 😐

How about using an overload set instead of default values? E.g. something like the following

```c++
sync_folder_items_result sync_folder_items(const folder_id& folder_id,
                                           int max_changes_returned = 512);

sync_folder_items_result sync_folder_items(const folder_id& folder_id,
                                           const std::string& sync_state,
                                           int max_changes_returned = 512);

sync_folder_items_result
sync_folder_items(const folder_id& folder_id, const std::string& sync_state,
                  const std::vector<item_id>& ignored_items,
                  int max_changes_returned = 512);

```